README.md

```diff
@@ -1,2 +1,13 @@
 # bazel_rules_hdl
 Hardware Description Language (Verilog, VHDL, Chisel, nMigen, etc) with open tools (Yosys, Verilator, OpenROAD, etc) rules for Bazel (https://bazel.build)
+ 
+# Project Vision
+"this" is a project that is designed to be a one stop shop for doing hardware dev with bazel.
+
+So that includes
```

Comment:
```suggestion
So that includes;

  * Supporting designs targeting FPGAs and ASIC.
  * Supporting software targeted to run on this custom hardware.
  * Supporting testing via simulation, formal verification, emulation and others.
```